rswag-ui/lib/rswag/ui/middleware.rb

```diff
@@ -14,7 +19,7 @@ def call(env)
         end
 
         if index_path?(env)
-          return [ 200, { 'Content-Type' => 'text/html', 'Content-Security-Policy' => csp }, [ render_template ] ]
+          return [ 200, { 'Content-Type' => 'text/html', CONTENT_SECURITY_POLICY_HEADER => csp }, [ render_template ] ]
```

Comment:
As `actionpack` is defined as [dependency](https://github.com/rswag/rswag/blob/master/rswag-ui/rswag-ui.gemspec#L25) of this gem, we can rely on the constants defined there

```suggestion
          return [ 200, { 'Content-Type' => 'text/html', ActionDispatch::Constants::CONTENT_SECURITY_POLICY => csp }, [ render_template ] ]
```
to get rid of the definition of `CONTENT_SECURITY_POLICY_HEADER` above

Reply:
Thanks @Lorel, made the adjustment!

Reply:
Hello, this is a good suggestion

However, as per gemspec, rswag-ui supports actionpack >= 5.2

https://github.com/rswag/rswag/blob/3013bcd9aaa73d1398bf8b0f4cece17efba91c63/rswag-ui/rswag-ui.gemspec#L25-L26

Constants were introduced 2 years ago, probably in 7.1.0

https://github.com/rails/rails/commit/1fd79abb549739c3db06f9fe197ff8836be639c0

I would reconsider this change

Reply:
Oh yeah, now I remember, I think that was why I didn't use ActionPack constant, due to this minimal version. @Lorel Are you okay with me reverting the last change?